build/gke-test-cluster/cluster.yml.jinja

```diff
@@ -34,6 +34,21 @@ resources:
               - https://www.googleapis.com/auth/devstorage.read_only
               - https://www.googleapis.com/auth/logging.write
               - https://www.googleapis.com/auth/monitoring
+        - name: "agones-system"
+          initialNodeCount: 1
+          config:
+            machineType: n1-standard-4
```

Comment:
Do we want 4 cpu, or would 2 do?

Reply:
At some point for scaling cluster of 10Ks you want bigger machines. For this one 2 is fine.